Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/repositories/MongoProblemRepository.java

```diff
@@ -167,7 +170,14 @@ public void patchProblem(int problemId, PatchProblemParams params) {
     @Override
     public boolean problemExists(int problemId) {
         return mongoTemplate.exists(query(where("_id").is(problemId)), ProblemData.class);
+    }
 
+    @Override
+    public boolean existsFile(String fileId) {
```

Comment:
should be a private method

`ProblemRepository` has nothing to do with the file